modules/planning/open_space/trajectory_smoother/dual_variable_warm_start_ipopt_interface.cc

```diff
@@ -115,15 +115,15 @@ bool DualVariableWarmStartIPOPTInterface::get_starting_point(
   // 1. lagrange constraint l, obstacles_edges_sum_ * (horizon_+1)
   for (int i = 0; i < horizon_ + 1; ++i) {
     for (int j = 0; j < obstacles_edges_sum_; ++j) {
-      x[l_index] = 0.5;
+      x[l_index] = 0.0;
```

Comment:
why change those init values? Did you get better performance with these start values?

Reply:
> why change those init values? Did you get better performance with these start values?

I think so, original point is inside current optimization problem's feasible set.